src/webextension/firstrun/components/welcome.js

```diff
@@ -20,6 +20,13 @@ export default function Welcome() {
         adipiscing elit. Etiam porta sem malesuada magna mollis euismod. Cras
         mattis consectetur purus sit amet fermentum.</p>
       </Localized>
+      <Localized id="welcome-warning">
+        <p style={{fontWeight: "bold"}}>Lorem ipsum dolor sit amet, consectetur.
```

Comment:
Let's not use `style`. We should do something like `<p><strong>` here instead. That should also help with blind users, since they might(?) hear something about `<strong>` but almost certainly won't hear anything about `style`.

Reply:
@jimporter Agreed on use of strong.

But after a few minutes of searching, I'm not sure how to make [Localized](https://github.com/projectfluent/fluent.js/wiki/Localized) take both the `p` and `strong` tags. It drops the `strong` once localized.

See my latest commits versus actual output when run:

![screen shot 2017-11-01 at 10 32 35 am](https://user-images.githubusercontent.com/49511/32285475-fedff7a0-beef-11e7-97b5-6f3237e9a57a.png)